### PR TITLE
[FRONT-279] Remove i18n from Marketplace

### PR DESCRIPTION
**Pull request by juhah**
This changeset turned out larger than I thought 😲 Anyway, removed Marketplace translations and `.po` file. Obviously no need to through all the changes but have a quick look and let me know if you spot any UI glitches.

### CODE REVIEW

File: app/src/marketplace/containers/EditProductPage/ConnectEthIdentity.jsx [hunk 1]
```diff
@@ -60,27 +59,27 @@ const ConnectEthIdentity = ({ className, disabled }: Props) => {
...
             )}
             {!walletLocked && isRequired && (
                 <Fragment>
-                    <Translate
-                        value="editProductPage.connectEthIdentity.identityNotLinked"
-                        tag="p"
-                        dangerousHTML
-                    />
+                    <p>
+                        You haven&#39;t connected your Ethereum address to your Streamr account yet.
```
Comment (mondoreale) [inline app/src/marketplace/containers/EditProductPage/ConnectEthIdentity.jsx:80]: `&#39;` → `&apos;`?

File: app/src/marketplace/containers/EditProductPage/ProductStreams.jsx [hunk 2]
```diff
@@ -46,16 +46,12 @@ const ProductStreams = ({ disabled }: Props) => {
...
-                <Translate
-                    tag="p"
-                    value="editProductPage.streams.description"
-                    docsLink={docsLinks.createProduct}
-                    dangerousHTML
-                />
+                <h1>Add streams</h1>
+                <p>
+                    Products can contain a range of streams, or a single &quot;firehose&quot; type stream, it&#39;s up to you.
+                    If you haven&#39;t made any streams yet, you can create one here. For help creating streams,
```
Comment (mondoreale) [inline app/src/marketplace/containers/EditProductPage/ProductStreams.jsx:52]: `&apos;` is pretty much `&#39;` but I find the first one easier to read as a human. ;)

File: app/src/marketplace/containers/ProductPage/usePurchase.js [hunk 3]
```diff
@@ -51,7 +50,7 @@ export default function usePurchase() {
         const { paymentCurrency, time, timeUnit, price } = accessPeriod || {}
 
         if (!accessPeriod || !time || !timeUnit || !paymentCurrency) {
-            throw new Error(I18n.t('no access period'))
```
Comment (mondoreale) [inline app/src/marketplace/containers/ProductPage/usePurchase.js:54]: Omg, guess we missed that one.

File: app/src/marketplace/containers/ProductPage/usePurchase.js [hunk 4]
```diff
@@ -78,7 +77,7 @@ export default function usePurchase() {
             )
 
             if (!purchasePrice) {
-                throw new Error(I18n.t('could not calculate price'))
```
Comment (mondoreale) [inline app/src/marketplace/containers/ProductPage/usePurchase.js:81]: And this one…

**Review by mondoreale — APPROVED** [review]
Looks like a hell of a lot of back&forth kind of work. Glad we're over this. 👏

**Review by tumppi — APPROVED** [review]
Nice! Could not find anything broken 👍

**Pull request merged**